packages/gatsby/src/redux/reducers/jobsv2.js

```diff
@@ -0,0 +1,38 @@
+module.exports = (
+  state = { active: new Set(), stale: new Map(), done: new Map() },
```

Comment:
`active` is not being used outside of this reducer (and then we only add and remove stuff from it), so we can drop it.

And about naming for `stale` and `done` "buckets":
> Ward:
> we do need a difference between done & active as we don't want to restart all done jobs
> pieh:
> sure - we need 2 buckets - done and “not done”
> honestly `notDone` is not a bad name to use here
> because `active` might suggest currently executing jobs, and `pending` might mean jobs waiting for start (if you squint a bit)
> Ward
> we could go for `unfulfilled` `fulfilled` or `incomplete` `complete`